### PR TITLE
ROC-2046 Use CCJ model

### PR DESCRIPTION
**Pull request by yarpo**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/ROC-2046

### Change description ###
Updated validation and mapping. 
Fixed mapping of RepaymentPlan model

**Does this PR introduce a breaking change?** (check one with "x")

```
[x] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/cmc/claimstore/models/ccj/PaymentSchedule.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package uk.gov.hmcts.cmc.claimstore.models.ccj;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public enum PaymentSchedule {
+    @JsonProperty("EACH_WEEK")
```
Comment (timja-kainos) [inline src/main/java/uk/gov/hmcts/cmc/claimstore/models/ccj/PaymentSchedule.java]: does this need to be here?

File: src/main/java/uk/gov/hmcts/cmc/claimstore/models/AmountRow.java [hunk 2]
```diff
@@ -6,13 +6,15 @@
...
 import java.math.BigDecimal;
 import java.util.Objects;
+import javax.validation.constraints.DecimalMin;
 
 @JsonInclude(JsonInclude.Include.NON_ABSENT)
 @ClaimantAmount
 public class AmountRow {
     private final String reason;
 
     @Money()
```
Comment (rlewan) [inline src/main/java/uk/gov/hmcts/cmc/claimstore/models/AmountRow.java]: Parentheses are not needed.

File: src/main/java/uk/gov/hmcts/cmc/claimstore/models/CountyCourtJudgment.java [hunk 3]
```diff
@@ -26,6 +27,7 @@
     private final TheirDetails defendant;
 
     @Money
+    @DecimalMin(value = "0.00")
     private BigDecimal paidAmount;
```
Comment (rlewan) [inline src/main/java/uk/gov/hmcts/cmc/claimstore/models/CountyCourtJudgment.java]: Missing `final`.

File: src/main/java/uk/gov/hmcts/cmc/claimstore/models/AmountRow.java (hunk 2 above)
Comment (yarpo) [reply src/main/java/uk/gov/hmcts/cmc/claimstore/models/AmountRow.java]: not my code, I'll remove

Comment (yarpo) [reply src/main/java/uk/gov/hmcts/cmc/claimstore/models/AmountRow.java]: done

File: src/main/java/uk/gov/hmcts/cmc/claimstore/models/CountyCourtJudgment.java (hunk 3 above)
Comment (yarpo) [reply src/main/java/uk/gov/hmcts/cmc/claimstore/models/CountyCourtJudgment.java]: done

**Pull request merged**